lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -653,3 +653,68 @@ integration builds, so the tests will initially fail when you submit a pull requ
 
 Once you're certain the failure is only due to the missing permissions, add a comment with the `ready_for_review`
 tag and explain that it's due to missing permissions.
+
+Your pull request will not be able to be merged until the tests are passing. If your pull request is failing due to missing permissions
+you can help with the process of getting your pull request into a mergeable state by collecting the minimum IAM permissions required to
+run the tests. The permissions used by CI are more restrictive than those in `hacking/aws_config/testing_policies` so this process is to create
+the most restrictive policy for the given tests.
+
+When constructing the policy the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_actions-resources-contextkeys.html)
+for the given service should be used. The [cloudonaut](https://iam.cloudonaut.io) documentation is sometimes a helpful cross-reference when troubleshooting.
+
+The simplest way is to run the tests using ample permissions with the `debug_botocore_endpoint_logs` module option for AnsibleAWSModule modules.
+After the `PLAY RECAP` of the completed tests the list of AWS actions that need to be allowed will be displayed.
+1) Ensure you set the `debug_botocore_endpoint_logs` module option set to True in your tests.
```

Comment:
This step is no longer needed now that it's integrated into the cloud module for ansible-test.